src/app/app.component.ts

```diff
@@ -30,7 +37,6 @@ export class AppComponent {
         docs: ComDocs,
         examples: ComExamples,
     };
-    public chosenItem = 1;
     public lists: IListItem[] = lists;
     public githubMaskLink = 'https://jsdaddy.github.io/ngx-mask/';
```

Comment:
is it make sense to make reusable constants in libraries. we have this link in site too